xml/System/Guid.xml

```diff
@@ -1581,11 +1701,11 @@
         <Parameter Name="format" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="2" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">When this method returns, the GUID as a span of characters.</param>
+        <param name="charsWritten">When this method returns, the number of characters written into the span.</param>
+        <param name="format">A read-only span containing the characters representing one of the following specifiers that indicates the exact format to use when interpreting <paramref name="input" />: "N", "D", "B", "P", or "X".</param>
+        <summary>Tries to format the current GUID instance into the provided span.</summary>
+        <returns><see langword="true" /> if the formatting was successful; <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the formatting operation was successful; <see langword="false" /> otherwise.</returns>
```